examples/optional-value.mjs

```diff
@@ -0,0 +1,55 @@
+// This is an example of adding support for an option with an optional value,
+// which can be used like a boolean-type or a string-type.
+
+import { parseArgs } from 'node:util';
+import process from 'node:process';
+
+const options = {
+  'host': {
+    type: 'string', short: 'h', default: 'default.com',
```

Comment:
```suggestion
    type: 'string',
    short: 'h',
    default: 'default.com',
```

Reply:
Will do, thanks. (I was annoyed the linter made me shorten the lines and didn't go all in!)